packages/demo/src/index.js

```diff
@@ -1,47 +1,111 @@
+                                    ]}
+                                />
+                            </Column>
+                            <Column>
+                                <Chart
+                                    chartType="donut"
+                                    subTitle="Pets"
+                                    title="100"
+                                    data={[{ x: 'Catss', y: cats }, { x: 'Dogs', y: dogs }, { x: 'Birds', y: birds }]}
+                                />
+                            </Column>
+                        </Section>
+                        <Section title="I am pie chart about pets">
+                            <Column>
+                                <Table
+                                    withHeader
+                                    rows={[
+                                        [ 'one', 'two' ],
+                                        [ 'one', 'two' ],
+                                        [ 'one', 'two' ]
+                                    ]}
+                                />
+                            </Column>
+                            <Column>
+                                <Chart
+                                    chartType="pie"
+                                    subTitle="Pets"
+                                    title="100"
+                                    data={[{ x: 'Catss', y: cats }, { x: 'Dogs', y: dogs }, { x: 'Birds', y: birds }]}
+                                />
+                            </Column>
+                        </Section>
+                        <Section title="I am just a table">
+                            <Column>
+                                <Table
+                                    withHeader
+                                    rows={[
+                                        [ 'one', 'two' ],
+                                        [ 'one', 'two' ],
+                                        [ 'one', 'two' ]
+                                    ]}
+                                />
+                            </Column>
+                            <Column/>
+                        </Section>
+                        <Section title="I am description list">
+                            <Column>
+                                <DescriptionList
+                                    rows={[
+                                        [ 'Mercury', 'Mercury (0.4 AU from the Sun) is the closest planet to the Sun and the smallest planet.' ],
+                                        [ 'Venus', 'Venus (0.7 AU) is close in size to Earth, (0.815 Earth masses) and like Earth, has a thick silicate mantle around an iron core.' ],
+                                        [ 'Earth', 'Earth (1 AU) is the largest and densest of the inner planets, the only one known to have current geological activity.' ]
+                                    ]}
+                                />
```

Comment:
That's not really how I would picture this, we should follow the regular browser `dl`, `dt` and `dd`.
```JSX
<Section title="I am description list">
  <Column>
    <DescriptionList>
      <DescriptionTerm>Mercury</DescriptionTerm>
      <DescriptionDetails>
        Mercury (0.4 AU from the Sun) is the closest planet to the Sun and the smallest planet.
      </DescriptionDetails>
      <DescriptionTerm>Venus</DescriptionTerm>
      <DescriptionDetails>
        Venus (0.7 AU) is close in size to Earth, (0.815 Earth masses) and like Earth, has a thick silicate mantle around an iron core.
      </DescriptionDetails>
      <DescriptionTerm>Earth</DescriptionTerm>
      <DescriptionDetails>
        Earth (1 AU) is the largest and densest of the inner planets, the only one known to have current geological activity.
      </DescriptionDetails>
    </DescriptionList>
  </Column>
</Section>
```

It's a bit more chatty, than the one you used, but this approach would make it easier for additional styling.